docs/conf.py

```diff
@@ -27,6 +27,7 @@
 extensions = [
     'sphinx.ext.todo',
     'sphinx.ext.autodoc',
+    'sphinx_rtd_theme',
```

Comment:
Oh interesting that this was missing, did it cause any issues?

Reply:
No issues, but if I understood correctly the [theme docs](https://sphinx-rtd-theme.readthedocs.io/en/latest/installing.html), the line is needed so the localized theme strings can be displayed correctly. We still don't have localized versions of the 0.13 docs, but we'll hopefully have French in a couple of months.

Reply:
Thanks for the explanation :)